redash/query_runner/databricks.py

```diff
@@ -35,6 +36,53 @@ def _build_odbc_connection_string(**kwargs):
     return ";".join([f"{k}={v}" for k, v in kwargs.items()])
 
 
+def split_sql_statements(query):
+    def strip_trailing_comments(stmt):
+        idx = len(stmt.tokens) - 1
+        while idx >= 0:
+            tok = stmt.tokens[idx]
+            if tok.is_whitespace or sqlparse.utils.imt(tok, i=sqlparse.sql.Comment, t=sqlparse.tokens.Comment):
+                stmt.tokens[idx] = sqlparse.sql.Token(sqlparse.tokens.Whitespace, ' ')
+            else:
+                break
+            idx -= 1
+        return stmt
+
+    def strip_trailing_semicolon(stmt):
+        idx = len(stmt.tokens) - 1
+        while idx >= 0:
+            tok = stmt.tokens[idx]
+            # we expect that trailing comments already are removed
+            if not tok.is_whitespace:
+                if sqlparse.utils.imt(tok, t=sqlparse.tokens.Punctuation) and tok.value == ";":
```

Comment:
What will happen if the query is like "SELECT * FROM A;;" Will originally invalid query become a valid query after we remove the last semicolon?

Reply:
There is a logic that just skips empty (including comment-only) statements, so this example will execute and return a result of `SELECT * FROM A`. Also there is a test case for it

Reply:
my point is that what will happen if we have two semicolon at the end of query. Will you remove one and make this invalid query become valid? I did not find such test case in you tests.